test/test_datapipe.py

```diff
@@ -1178,6 +1178,17 @@ def fn_n1(d0, d1):
         def fn_nn(d0, d1):
             return -d0, -d1, d0 + d1
 
+        def fn_n1_def(d0, d1=1):
+            return d0 + d1
+
+        def fn_n1_kwargs(d0, d1, **kwargs):
+            return d0 + d1
+
+        def fn_n1_pos(d0, d1, *args):
+            return d0 + d1
+
```

Comment:
Could you please add more testing functions with signature like?
- `def fn_xxxxxx(d0, d1=1, *args, d2, **kwargs)`
- `p_fn = partial(fn_xxxxxx, d2=2)`